powerauth-java-crypto/src/test/java/io/getlime/security/powerauth/crypto/encryption/BasicEciesEncryptorTest.java

```diff
@@ -268,10 +268,17 @@ public void testKdf() {
 
     /**
      * Test that values match client side test vectors.
+     *
+     * <h5>PowerAuth protocol versions:</h5>
+     * <ul>
+     *     <li>2.0</li>
+     *     <li>2.1</li>
+     * </ul>
+     *
      * @throws Exception When test fails.
      */
     @Test
-    public void testVectors() throws Exception {
+    public void testVectorsV2() throws Exception {
```

Comment:
I think that we don't need to split this into v2 & v3. At the fundamental level, v3 is simply superset of v2. If the sharedInfo1 parameter is empty, then the computation is equal to v2. The new vectors already covers both situations.

You can use that old data if you want, there's nothing wrong with that, but we don't need to highlight that it's a different version. We will not have two different ECIES implementations in the code. Just one, with right parameters for both, v2 & v3 protocols.

Reply:
Ok, I will clean it up.